src/_gettsim/einkommensteuer/einkünfte/aus_kapitalvermögen/aus_kapitalvermögen.py

```diff
@@ -7,21 +7,25 @@
 
 @policy_function(end_date="2008-12-31", leaf_name="betrag_y")
 def betrag_y_mit_sparerfreibetrag_und_werbungskostenpauschbetrag(
-    kapitalerträge_y: float,
+    einnahmen__aus_kapitalvermögen__kapitalerträge_y: float,
```

Comment:
```suggestion
    einnahmen__kapitalerträge_y: float,
```
Would seem clear enough?